orga/tests/acceptance/authentication-test.js

```diff
@@ -305,6 +305,28 @@ module('Acceptance | authentication', function(hooks) {
           assert.dom('.sidebar-menu a:first-child').hasNoClass('active');
         });
 
+        module('When featureToggle isCertificationResultsInOrgaEnabled is enabled', function() {
```

Comment:
Proposition de formulation plus "métier" :
"When the FT_IS_CERTIFICATION_RESULTS_IN_ORGA_ENABLED feature toggle is enabled"